### PR TITLE
Add a config file option (#230)

### PR DESCRIPTION
**Pull request by jucrouzet**
Add a `--config` option in CLI to specify options in a config file (See #230)

### CODE REVIEW

**Comment by jucrouzet** [comment]
IMHO, seemed a good options, especially when automating tests in Gulp/Grunt/etc. where we can change a very long command to a simple one.

File: bin/selenium-standalone [hunk 1]
```diff
@@ -27,17 +28,33 @@ which('java', function javaFound(err, javaPath) {
...
+        throw new Error('Config file does not exports an object');
+      }
+    } catch(err) {
+      console.log('Error parsing config file :', err);
+      return;
+    }
+  }
+
+  // Merge default options, command line options and options from config file
+  options = merge({}, defaultConfig, options, configFromFile);
```
Comment (stephenash) [inline bin/selenium-standalone]: `options` and `configFromFile` seem backwards to me. I would expect that CLI args have precedence over everything else.

File: README.md [hunk 2]
```diff
@@ -40,9 +40,36 @@ selenium-standalone start -- -role hub
...
+selenium-standalone install --config=./config/seleniumConfig.js
 
 ```
 
+Config file can be a JSON file or a [module file](https://nodejs.org/api/modules.html#modules_file_modules) that exports options as an object.
+
+JSON example :
+```json
+{
+  basePath: "/tmp/"
```
Comment (stephenash) [inline README.md]: `basePath` should be in quotes to be valid JSON

**Comment by stephenash** [comment]
Had a few small comments about the code, but overall looks like a good direction. Can you please add some unit tests as well, especially to validate that options are being overridden as expected?

**Comment by vvo** [comment]
Nice! What's your take on JSON versus JavaScript configuration files? I do prefer JavaScript configuration files, most of the time it's more flexible. So I wonder: should we support JSON and JavaScript or just JavaScript and wait for someone to complain with a usecase?

File: bin/selenium-standalone (hunk 1 above)
Comment (jucrouzet) [inline bin/selenium-standalone]: That's right, will reverse the arguments.

File: README.md (hunk 2 above)
Comment (jucrouzet) [inline README.md]: Yes... Thank you, will correct it.

**Comment by jucrouzet** [comment]
@vvo : I personnaly prefer JS file also, but as `require()` accepts both, thought it would be better to leave it untouched. We can just use a JS example in the README

**Comment by vvo** [comment]
> I personnaly prefer JS file also, but as require() accepts both, thought it would be better to leave it untouched. We can just use a JS example in the README

Works for me

**Comment by jucrouzet** [comment]
Changes done.

**Comment by vvo** [comment]
LGTM

**Comment by vvo** [comment]
One of you guys want to do the release? That would help!

**Comment by vvo** [comment]
Sorry @jucrouzet can you reopen your PR and add some tests as suggested? thx

**Comment by jucrouzet** [comment]
@vvo Yup, I saw it. I'm in a rush at work right now, but will do it probably tomorrow.

**Comment by vvo** [comment]
Ok just wanted to make sure you were not too much pissed-off by how it went. No need to rush

**Pull request merged**